### PR TITLE
Docs: Improve capitalization, style, and formatting in metrics

### PR DESCRIPTION
**Pull request by Simran-B**
Backport of https://github.com/arangodb/arangodb/pull/18385

### CODE REVIEW

File: Documentation/Metrics/rocksdb_compaction_pending.yaml [hunk 1]
```diff
@@ -11,6 +11,6 @@ exposedBy:
   - agent
   - single
 description: |
-  This metric exhibits the RocksDB metric "compaction-pending".
+  This metric exhibits the RocksDB metri`compaction-pending`.
```
Comment (nerpaula) [inline Documentation/Metrics/rocksdb_compaction_pending.yaml:14]:
```suggestion
  This metric exhibits the RocksDB metric `compaction-pending`.
```

File: Documentation/Metrics/rocksdb_compression_ratio_at_level1.yaml [hunk 2]
```diff
@@ -14,4 +14,4 @@ description: |
   This metric exhibits the compression ratio of data at level 1 in RocksDB's
   log structured merge tree. Here, compression
   ratio is defined as uncompressed data size / compressed file size.
-  Returns "-1.0" if there are no open files at level 1.
+  Return`-1.0` if there are no open files at level 1.
```
Comment (nerpaula) [inline Documentation/Metrics/rocksdb_compression_ratio_at_level1.yaml:17]:
```suggestion
  Returns `-1.0` if there are no open files at level 1.
```

File: Documentation/Metrics/arangodb_scheduler_queue_time_violations_total.yaml [hunk 3]
```diff
@@ -15,14 +15,13 @@ exposedBy:
...
   Number of tasks/requests dropped because the client-specified queue time
-  requirements, as indicated by client applications in the request header
-  "x-arango-queue-time-seconds" could not be satisfied by the receiving 
-  server instance. This happens when the actual time need to queue/dequeue
-  requests on the scheduler queue exceeds the maximum time value that the
-  client has specified in the request. 
-  Whenever this happens, the client application will get an HTTP 412 error 
+  requirements, as indicated by client applications in the
+  `x-arango-queue-time-seconds` HTTP request header could not be satisfied by
+  the receiving server instance. This happens when the actual time need to
```
Comment (jsteemann) [inline Documentation/Metrics/arangodb_scheduler_queue_time_violations_total.yaml:20]:
```suggestion
  the receiving server instance. This happens when the actual time needed to
```

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz/view/Documentation/job/arangodb-ANY-examples/3000/

**Pull request merged**